webviews/editorWebview/index.css

```diff
@@ -20,9 +21,11 @@
 }
 
 #sidebar {
+	display: flex;
+	flex-direction: column;
+	gap: 16px;
 	grid-column: 2;
 	grid-row: 2;
-	padding-left: 20px;
 }
 
 a:focus,
```

Comment:
Please add `.icon-button:focus` to the list bellow because right now, there is no outline visible for icon buttons. To fix this, you need to add the class `icon-button` to every icon button as well.

Reply:
```
.section .icon-button:hover,
.section .icon-button:focus {
	background-color: var(--vscode-toolbar-hoverBackground);
}

.section .icon-button:focus {
	outline: 1px solid var(--vscode-focusBorder);
	outline-offset: unset;
}
```